examples/js/loaders/deprecated/LegacyGLTFLoader.js

```diff
@@ -1634,6 +1634,30 @@ THREE.LegacyGLTFLoader = ( function () {
 									geometry.addAttribute( 'skinIndex', bufferAttribute );
 									break;
 
+								default:
+
+									if ( primitive.material ) {
+
+										var material = json.materials[ primitive.material ];
+
+										if ( material.technique ) {
+
+											var parameters = json.techniques[ material.technique ].parameters;
+
+											for( var attributeName in parameters ) {
+
+												if ( parameters [ attributeName ][ 'semantic' ] === attributeId ) {
+
+													geometry.addAttribute( attributeName, bufferAttribute );
+
+												}
+
+											}
+
+										}
+
+									}
```

Comment:
One more nit (sorry!), rather than nesting these could you use `break`? Example:

```js
if ( ! primitive.material ) break;

var material = ...;

if ( ! material.technique ) break;

var parameters = json.techniques[ material.technique ].parameters || {};

...
```

Reply:
No problem! And yeah, TMP file, whoops. I'll get rid of that when I make the changes above.